stable/kube-state-metrics/values.yaml

```diff
@@ -1,4 +1,12 @@
 # Default values for kube-state-metrics.
+global:
```

Comment:
Shouldn't this be on the root level instead of under `global`?

Reply:
Actually, as it is used with prometheus-operator and prometheus has it under global, it will avoid setting this twice